### DISCUSSION TITLE
Can others get a Galaxy Brain Badge from answering this post p2?

### DISCUSSION THREAD

**Discussion by peterdanwan** [peterdanwan/gimme_readme#20] [Q&A] [answered]
Can others get a Galaxy Brain Badge from answering this post p2?

Please post a screenshot after I accept your answer, showing if you got the badge!

**Comment by Mimi-Dang** [comment, marked as answer]
Hi, Peter, I am here again to get the badge, would give you a screenshot when my answer get marked.

**Reply by Mimi-Dang** [reply]
![image](https://github.com/user-attachments/assets/19d2266e-92dd-4918-a793-d2cbd7fb9425)

**Reply by peterdanwan** [reply]
Perfect, well done! I'm closing this discussion.
